src/main/webapp/help/parasoftCoverage/help-referenceJob.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    Specify a reference job to use if "Quality gates" is configured. To use the current job, leave this field
```

Comment:
Change the message after "-" as:
which includes the names of all items in the hierarchy from the root Jenkins instance to the job, separated by '/'."

Reply:
The message is updated.

There's also some docs in Jenkins User Handbook.

[Referencing another project by name](https://www.jenkins.io/doc/book/using/referencing-another-project-by-name/#referencing-components-inside-projects)

Reply:
According to what Alfie said, supporting the "relative paths" feature mentioned in the link requires extra work, but at the moment, we don't know the solution. I believe we don't need to align with all the methods supported by Jenkins. Users using the full name should be sufficient unless there are user complaints.
@rwang-parasoft Please merge this pull request if you don't have any other concerns.